website/src/components/home/Hero.vue

```diff
@@ -28,31 +33,46 @@
       Chakra UI Vue Next is a simple modular and accessible component library
       that gives you the building blocks to build Vue applications with speed.
     </chakra.p>
-    <CFlex flex-dir="row">
+    <CFlex
+      flex-dir="row"
+      :justify-content="{ base: 'center', sm: 'flex-start' }"
+    >
       <CButton
         as="a"
         text-decoration="none"
         href="https://github.com/chakra-ui/chakra-ui-vue-next"
         size="lg"
         target="_blank"
+        rel="noopener noreferrer"
         color-scheme="green"
+        bg="vue.600"
       >
         Get started
       </CButton>
       <CButton
         as="a"
         text-decoration="none"
         target="_blank"
+        rel="noopener noreferrer"
         href="https://github.com/chakra-ui/chakra-ui-vue-next"
         left-icon="github"
         ml="4"
         variant="outline"
         color-scheme="green"
         size="lg"
+        color="vue.600"
       >
         Github
       </CButton>
     </CFlex>
+    <chakra.img
+      src="/src/assets/chakra-ui-vue-logo.svg#svgView(viewBox(0,0,150,190))"
```

Comment:
The `:src` attribute should begin with the `/assets/` directory since it gets built into the public directory. That's why it probably wasn't showing in the deploy preview. Try running the build command for docs to see which directory it gets built to.